app/cdap/components/Replicator/Create/Content/SelectColumnsWithTransforms/TransformAdd.tsx

```diff
@@ -135,7 +135,10 @@ export default function TransformAddButton({
         <MenuItem onClick={handleMenuClick}>
           Rename <ArrowRight />
         </MenuItem>
-        <MenuItem onClick={handleMaskOpen}>
+        <MenuItem
+          disabled={!javaSqlStringTypes.includes(row.type.toLowerCase())}
```

Comment:
Could we extract out this condition check as a reusable function instead of repeat the code four times?